api.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+func btcctlCommand(args []string) {
+	argsLen := len(args)
+	if argsLen == 0 {
+		fmt.Println("No commands were provided")
+		return
+	}
+
+	btcctlConfPath := "/Users/draku/go/src/orcacointest/cmd/btcctl/sample-btcctl.conf"
+	argIdx := 0
+	for argIdx < argsLen {
+		arg := args[argIdx]
+		switch arg {
+		case "getbalance":

```

Comment:
Define the command strings as enumerators or constants and utilize them instead of hardcoding the strings here. This approach simplifies the renaming process in the future, as you only need to update the definition. Currently, if you wish to rename a command, you would need to modify multiple lines (Eg: line 20, 21, and 24).

Reply:
Fixed